[v4] Snippet for community contributions for integration guides (#496)
strapi/documentation: docs/developer-docs/latest/developer-resources/content-api/snippets/integration-guide-not-updated.md

::: caution

> This third-party integration guide might not be up-to-date with Strapi v4. [Contributions](https://github.com/strapi/documentation/blob/main/CONTRIBUTING.md) are most welcome.

Author: ```suggestion
This third-party integration guide might not be up-to-date with Strapi v4. [Contributions](https://github.com/strapi/documentation/blob/main/CONTRIBUTING.md) are most welcome 🤗.
```

> This third-party integration guide might not be up-to-date with Strapi v4. We encourage you to help us updating it by [contributing to our documentation repository on GitHub](https://github.com/strapi/documentation/blob/main/CONTRIBUTING.md).

Author: ```suggestion
This third-party integration guide might not be up-to-date with Strapi v4. [Contributions](https://github.com/strapi/documentation/blob/main/CONTRIBUTING.md) are most welcome.
```
